### PR TITLE
TFW-3751 - Add inline toggle to input

### PR DESCRIPTION
**Pull request by tVeratti**

### CODE REVIEW

File: components/form/iconPopout.beta.scss [hunk 1]
```diff
@@ -19,6 +19,10 @@ $arrow-size: 4px;
 
         .icon-popout__button:after { @include arrow($arrow-size, $color-white, 'up'); }
     }
+
+    &--disabled {
```
Comment (jasonhale) [inline components/form/iconPopout.beta.scss]: This definitely works.  Would it be better to override the cursor at the button instead?
```css
.icon-popout__button {
  ...
  &:disabled {
    cursor: default;
  }
}
```

Didn't know if there were other issues involved here.

Comment (tVeratti) [reply components/form/iconPopout.beta.scss]: I think I was planning to do more with this but decided restyling the popout would probably fit better as its own styling update story.

Comment (jasonhale) [reply components/form/iconPopout.beta.scss]: Yeah.  Ok.  This is odd to me, but it works!  Thanks

File: components/form/input/input.beta.scss [hunk 2]
```diff
@@ -143,3 +143,22 @@
...
+
+    .input__label {
+        margin-bottom: 0;
+    }
+
+    .input__wrap {
+        flex: 1;
+    }
+
+    // Toggle
```
Comment (troyblank) [inline components/form/input/input.beta.scss]: is this really for the toggle? I never would have guessed :P ... I would consider removing this redundant comment in seriousness but not a show stopper.

File: components/form/toggle.beta.scss [hunk 3]
```diff
@@ -32,45 +32,45 @@
     }
 
     :checked ~ div {
-        .nob {
+        .toggle__nob {
```
Comment (troyblank) [inline components/form/toggle.beta.scss]: awesome thanks for making old stuff up to snuff.

**Comment by jasonhale** [comment]
@troyblank - that's biz rule is handled by work I'm doing that builds off this.  PR coming soon.

**Pull request merged**